wallet/src/customized/java/com/android/identity_credential/wallet/WalletApplicationConfiguration.kt

```diff
@@ -24,6 +24,12 @@ object WalletApplicationConfiguration {
      */
     const val WALLET_SERVER_DEFAULT_URL = "https://ws.example.com/server"
 
+    /**
+     * Unless [WALLET_SERVER_DEFAULT_URL] is set to "dev:", set it to the same value as
```

Comment:
nit: I'm a little confused by this comment. It doesn't explain what MIN_SERVER_DEFAULT_URL is, just describes what to set it to. Maybe add a sentence briefly describing how this is used?

Reply:
Yeah, this is this weird customization framework... I referenced the main version of this file, which is authoritative.